utilities/getProcessedImage.ts

```diff
@@ -16,7 +19,7 @@ import { getMaskedAsset } from './getMaskedAsset';
 
 export const getProcessedImage = (
   imageSrc: string = '',
-  crop: string = '',
+  crop: string = getSbImageSize(imageSrc)?.width > 2000 ? '2000x0' : '',
```

Comment:
Made a small change to getProcessedImage utility, so that If a crop dimension is not provided and if the image width is greater than 2000px, the crop dimension will be set to "2000x0" to keep the original aspect ratio and downsize the image to 2000px wide. I don't think there is a reason to have any images on the site that is larger than 2000px wide at the moment.

Reply:
Smart.